src/injected/visualization/get-visualization-instances-for-tab-stops.ts

```diff
@@ -64,5 +65,23 @@ export const GetVisualizationInstancesForTabStops = (
         });
     });
 
+    forOwn(
```

Comment:
Why not just do this when we create the instances in the previous loop?

Thinking about this now ... I was wondering what is the hierarchy for the selected TabbedItemType. I think we don't need to care about it _for now_ because as long as we show an error, it is fine (and the scenario where we have a missing tabbed issue AND another error would need to be manually manufactured).

Reply:
Yeah, I was thinking about the same thing. I arbitrarily chose to have the missing item config take precedence over errored, but that can easily be changed.